R/RunDiagnostics.R

```diff
@@ -566,7 +566,13 @@ getConceptSets <- function(cohorts) {
     
     sqlParts <- SqlRender::splitSql( gsub("with primary_events.*", "", cohort$sql))
     sqlParts <- sqlParts[-1]
-    conceptSetIds <- as.numeric(gsub("^.*SELECT ([0-9]+) as codeset_id.*$", "\\1", sqlParts, ignore.case = TRUE))
+    conceptSetIds <- stringr::str_extract_all(string = sqlParts, 
```

Comment:
I don't understand how this is an improvement. I find the new code much harder to read, it is longer, and
- Does not require 'codeset_id' to be present, so the extracted ID might be something else?
- Is not case-sensitive, so could more easily break down. 

Please do not simply convert any syntax you are unfamiliar with with tidyVerse. I recommend reading [a primer on RegEx](http://marvin.cs.uidaho.edu/Handouts/regex.html) to learn more about this powerful tool.